posthog/utils.py

```diff
@@ -333,13 +331,10 @@ def render_template(template_name: str, request: HttpRequest, context: Dict = {}
 
     context["posthog_app_context"] = json.dumps(posthog_app_context, default=json_uuid_convert)
 
-    if not posthog_distinct_id:
-        posthog_distinct_id = str(uuid.uuid4())
-
-    feature_flags = posthoganalytics.get_all_flags(posthog_distinct_id, only_evaluate_locally=True)
-    posthog_bootstrap["distinctID"] = posthog_distinct_id
```

Comment:
just double checking distinctID doesn't need to be set now?

Reply:
For us _specifically_, no, because our frontend calls identify with this distinctID on load always: https://github.com/PostHog/posthog/blob/master/frontend/src/scenes/userLogic.ts#L99

and we depend on this to update the distinctID.

For any random other app, this might not be the case

Reply:
I'll edit docs to clarify this as well